osf/metadata/serializers/json_ld.py

```diff
@@ -0,0 +1,66 @@
+from osf.metadata import gather
+from osf.metadata.serializers import _base
+
+
+class JsonLdMetadataSerializer(_base.MetadataSerializer):
+    mediatype = 'text/json-ld'
+
+    def filename(self, osfguid: str):
+        return f'{osfguid}-metadata.json-ld'
+
+    def serialize(self, basket: gather.Basket):
+        node = basket.focus.dbmodel
```

Comment:
something that seemed good about `osf.metadata` was a clear separation of concerns: the serializers know only osf's uri-based metadata model, but are completely unaware of the specifics of osf's data layer.

pls either get the metadata from the `basket` or reimplement this someway else that doesn't call `pls_gather_item_metadata` unnecessarily